### DISCUSSION TITLE
IOCB as academic institution

### DISCUSSION THREAD

**Discussion by jmkormanik** [RosettaCommons/rosetta#66] [ROSIE] [answered]
Hello,
I'd like to access ROSIE but I'm getting an error that I have to confirm my institution is academic. I currently work at IOCB Prage, https://uochb.cz/en, which is a part of the Czech Academy of Sciences, hence the e-mail domain (uochb.cas.cz). A similar domain can also be used for the same institute (iocb.cas.cz), so it would be great if both got marked as academic institutions.
Thank you and have a nice day,
Ján Michael Kormaník

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know! I added both domain so it should work now. Please let me know if you need further help.

**Reply by huytont-mpi** [reply]
could you please add MPINAT
https://www.mpinat.mpg.de/de
email@mpinat.mpg.de

**Reply by lyskov** [reply]
This is done! Please try to login again. Thanks,
